### PR TITLE
Fixing The Chain Ring Display & Adding some Improvements

### PR DESCRIPTION
**Pull request by AtelyPham**
## Summary of changes
- Fixing the bug about The ChainsRing component not displaying the icon and label properly in case we only have the destination chain.
- The chain list extends to the bottom of the card.
- Unavailable tokens must be disabled.
- "Try another wallet" should open the connect wallet modal
- The AmountInput only allows number value

### Reference issue to close (if applicable)
- Closes https://github.com/webb-tools/webb-dapp/issues/775
- A part of the issue https://github.com/webb-tools/webb-dapp/issues/749

-----
### Code Checklist 

- [x] Tested
- [x] Documented

### CODE REVIEW

File: apps/bridge-dapp/src/hooks/useTryAnotherWalletWithView.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            onWalletSelect={async (wallet) => {
+              await switchWallet(chain, wallet);
+            }}
+            onClose={() => setIsModalOpen(false)}
+            connectingWalletId={connectingWalletId}
+            failedWalletId={failedWalletId}
+            onTryAgainBtnClick={async () => {
+              if (!selectedWallet) {
+                throw new Error(
+                  'There is not selected wallet in try again function'
```
Comment (dutterbutter) [inline apps/bridge-dapp/src/hooks/useTryAnotherWalletWithView.tsx:36]:
```suggestion
                  'There is no wallet selected. Please select a wallet and try again.'
```

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Small proposed changes.

**Comment by AtelyPham** [comment]
Add more improvements:

- Consistent all the small all-cap text font size
- Disabled amount inputs (custom and fixed) when token or chain is not selected
- Do not pop up the wallet modal when changing the chain if the selected chain is supported by the current active wallet.

**Review by dutterbutter — APPROVED** [review]
LGTM @AtelyPham ! Since it addresses comments from #749 let's wait to merge to get visual reviews from @seeddisperser or @monaiuu

**Review by AhmedKorim — APPROVED** [review]

**Comment by seeddisperser** [comment]
One more fix: 

if the user has not connected their wallet, the button at the bottom of the transfer and withdraw tabs should prompt the user to connect their wallet, as implemented in the deposit tab.

**Comment by AtelyPham** [comment]
Moved remaining items to issue https://github.com/webb-tools/webb-dapp/issues/749.

**Pull request merged**